### PR TITLE
Auto-update URL, Fix vchip undefined bug, page reload on rom change for other fixes

### PR DESCRIPTION
**Pull request by zoul0813**
* fix vchip undefined bug
* remove unused anchor hrefs, styled anchors
* refactored readrom.js
* added auto-updating url on rom selection
  - does not change "latest" or "nightly", to allow for bookmarking
  - forces reload on rom change, after confirmation (prevents bugs related to restart/reset/etc... start each new rom image "fresh")

### CODE REVIEW

File: view/readrom.js [hunk 1]
```diff
@@ -110,34 +91,55 @@ const prebuilt_json_url_path = "/roms/index.json";
...
+    });
+    const stable  = index.stable.map((entry) => to_option(entry, false));
 
+    let all_options = [
+        `<option value="">Choose an image...</option>`,
+    ];
+    if(index.latest) {
+        all_options.push(`<optgroup label="--- Latest ---" data-type="latest">` + latest + `</optgroup>`);
+    }
+    if(index.nightly) {
```
Comment (Zeal8bit) [inline view/readrom.js:130]: An index is unlikely to belong to `nightly`, `latest` and `stable`, so using `else if` seems more appropriate

**Review by Zeal8bit — COMMENTED** [review]
Looks good to me! Please fix the merge conflict first to that I can merge it

File: view/readrom.js (hunk 1 above)
Comment (zoul0813) [reply view/readrom.js:130]: Not sure I understand - “index” here is the object that holds the latest, nightly and stable properties

these checks are just making sure those properties exist before attempting to access them

Comment (Zeal8bit) [reply view/readrom.js:142]: My bad, ignore my comment, I misunderstood it

**Pull request merged**